### PR TITLE
Merge all the common template parts into a base

### PR DESCRIPTION
**Pull request by edds**
Lots of the templates use the same base template. Merge them into a
single base page which everything can then extend from. This would make
updating global page layouts much easier.

This will let us iterate aspects of all pages in one go. Like using a component
for all the titles.

### CODE REVIEW

File: app/views/root/_base_page.html.erb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+      <div class="inner">
+        <%= yield %>
+      </div>
+    </article>
+    <% json_link ||= publication_api_path(publication, :edition => edition)  %>
+    <%= render 'publication_metadata', :publication => publication, :api_links => { 'application/json' => json_link } %>
+  </div>
+</main>
+
+<% if !local_assigns.include?(:related_items) || related_items != false %>
```
Comment (dsingleton) [inline app/views/root/_base_page.html.erb]: This is quite a confusing conditional, it's hard to understand whats it's doing at a glance, could you make it a bit easier to understand?

It looks like it should be a guard against `:related_items` being set, but doesn't seem to be?

**Comment by dsingleton** [comment]
Weird `if` aside this looks good, and for other reviewers it's best to check the [diff with whitespace ignored](https://github.com/alphagov/frontend/pull/740/files?w=1), much more readable :+1:

File: app/views/root/_base_page.html.erb (hunk 1 above)
Comment (edds) [inline app/views/root/_base_page.html.erb]: Fair point. I have updated to use a `fetch` with a default rather than two negatives. Is that better?

**Comment by dsingleton** [comment]
`show_related_items` reads a lot better, LGTM.

**Pull request merged**